### PR TITLE
Release/1.1.1

### LINKED ISSUE #99: Release version 1.1.1
This issue is for tracking changes for the 1.1.1 release. Target release date: 19 Dec 2022.

## [Release steps](https://github.com/10up/convert-to-blocks/blob/develop/CONTRIBUTING.md#release-instructions)

- [x] Branch: Starting from `develop`, cut a release branch named `release/1.1.1` for your changes.
- [x] Version bump: Bump the version number in `config.php`, `convert-to-blocks.php`, `readme.txt`, `package.json` and `package-lock.json` if it does not already reflect the version being released.  Update both the plugin "Version:" property in `convert-to-blocks.php` and the plugin `$plugin_version` constant in `config.php`.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Readme updates: Make any other readme changes as necessary.  `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk`, ensuring you pull the most recent changes into `develop` first (`git checkout develop && git pull origin develop && git checkout trunk && git merge --no-ff develop`). `trunk` contains the stable development version.
- [x] Push: Push your `trunk` branch to GitHub (e.g. `git push origin trunk`).
- [x] [Compare](https://github.com/10up/convert-to-blocks/compare/trunk...develop) trunk to develop to ensure no additional changes were missed.
- [x] Test the pre-release ZIP locally by [downloading](https://github.com/10up/convert-to-blocks/actions/workflows/build-release-zip.yml) it from the Build release zip action artifact and installing it locally. Ensure this zip has all the files we expect, that it installs and activates correctly and that all basic functionality is working
- [x] Release: Create a [new release](https://github.com/10up/convert-to-blocks/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch.  Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the [closed issues on the milestone](https://github.com/10up/convert-to-blocks/milestone/7?closed=1).  The release should now appear under [releases](https://github.com/10up/convert-to-blocks/releases).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/convert-to-blocks/actions) to finish deploying to the WordPress.org repository.  If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/convert-to-blocks/.  This may take a few minutes.
- [x] Close the milestone: Edit the [milestone](https://github.com/10up/convert-to-blocks/milestone/7) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.1.1` do not make it into the release, update their milestone to `1.2.0`, or `Future Release`

### PR DESCRIPTION
**Pull request by dkotter**
### Description of the Change

Release and version bump to 1.1.1.

Closes #99 

### How to test the Change

Verify manually via VS Code or the GitHub UI

### CODE REVIEW

**Comment by jeffpaul** [comment]
@dkotter seeing a bunch of linting issues.  Feel like those should be resolved here or handled as a separate PR in the next release?

**Review by jeffpaul — DISMISSED** [review]
Looks good, sorry for the slow review, fine to release and then update the date in CHANGELOG.md/readme.txt post-release... thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased] - TBD
 
+## [1.1.1] - 2022-12-19
```
Comment (dkotter) [inline CHANGELOG.md]:
```suggestion
## [1.1.1] - 2023-01-05
```

File: readme.txt [hunk 2]
```diff
@@ -43,6 +43,24 @@ Nested / Inner Block support does not work with Gutenberg bundled with WordPress
 
 == Changelog ==
 
+= 1.1.1 - 2022-12-19 =
```
Comment (dkotter) [inline readme.txt]:
```suggestion
= 1.1.1 - 2023-01-05 =
```

**Pull request merged**